schemas/event.yml

```diff
@@ -68,19 +69,31 @@
             It is used to represent events that indicate that something happened.
         - name: metric
           description: >
-            This value is used to indicate that this event that a numeric measurement
-            was taken at given point in time.
+            This value is used to indicate that this event describes a numeric measurement
```

Comment:
same as above - add periodic in to differentiate a periodic measurement of some value vs. a state at a given time

Reply:
Again, both state and metric event kinds can occur at a fixed period, or on an ad hoc, or irregular basis.

Reply:
good with it here too